programs/survival/lava_sponge.sc

```diff
@@ -10,7 +10,7 @@ __config() -> (
 
 
 __on_player_places_block(player, item_tuple, hand, block)->(
-    if(hand == 'mainhand' || item_tuple:0 == 'sponge' || all(neighbours(pos(block)),_=='lava'),
+    if(hand == 'mainhand' && item_tuple:0 == 'sponge' && neighbours(pos(block))~'lava' != null,

```

Comment:
You could use the suggestion I made here: https://github.com/gnembon/scarpet/pull/220#discussion_r657917284
It appears that you half-implemented it, i.e you changed the whole thing to be nested in an if-loop, but then didn't copy the exact boolean check condition I wrote: `hand=='mainhand' && item_tuple:0=='sponge' && any(neighbours(pos(block)), _=='lava')`

Reply:
Agh nvm Ill do it mself